include/irr/asset/IAsset.h

```diff
@@ -259,7 +266,12 @@ class SAssetBundle
 		//! Overloaded operator checking if both collections of Assets\b are\b the same arrays in memory
 		bool operator==(const SAssetBundle& _other) const
 		{
-			return _other.m_contents == m_contents;
```

Comment:
explain why are we changing this?

Reply:
ok, I remember https://github.com/buildaworldnet/IrrlichtBAW/pull/410#discussion_r362123335 is that the reason?

Reply:
no, it's fix so that it actually compare contents (contained assets) and not just array pointers

Reply:
hmm ok, my original algorithm assumed that no one would be so dumb as to add the same contents twice... but I guess making it more idiot-proof doesn't cost that much perf, so OK.